tests/functional/test_retrieval.py

```diff
@@ -3,28 +3,95 @@
 import numpy as np
 import pytest
 import torch
+from pytorch_lightning import seed_everything
```

Comment:
don't use any PL features in tests, for seeding use `torch.manuall_seed` and `torch.cuda.manuall_seed_all`